frontend/src/container/FeedItemWithPreview.jsx

```diff
@@ -220,13 +340,25 @@ export class FeedItemWithPreview extends React.Component {
                 shouldScrollToBottom={false}
                 showInvalidMentionPopup={state.showInvalidMentionPopupInComment}
                 showTitle={false}
-                timelineData={props.commentList}
+                timelineData={this.getTimelineData()}
                 wysiwyg={state.timelineWysiwyg}
                 onClickCancelSave={this.handleCancelSave}
                 onClickOpenFileComment={this.handleClickOpenFileComment}
                 onClickSaveAnyway={this.handleClickValidateAnyway}
                 searchForMentionInQuery={this.searchForMentionInQuery}
                 workspaceId={props.workspaceId}
+                onClickTranslateComment={(
+                  comment => handleTranslateComment(
+                    FETCH_CONFIG.apiUrl,
+                    props.content.workspaceId,
+                    props.content.id,
+                    comment.content_id,
+                    props.i18n.language,
+                    props.system.config,
+                    this.commentSetState.bind(this, comment.content_id)
```

Comment:
is it required to bind this here?

Reply:
I am using bind to set the first comment, `this` is kind of irrelevant here but it is a necessary argument.

Reply:
I guess I can write this as:

```suggestion
                    () => this.commentSetState(comment.content_id)
```